server/services/storage/agent_framework_storage_service.ts

```diff
@@ -193,23 +197,21 @@ export class AgentFrameworkStorageService implements StorageService {
     try {
       const response = (await this.client.transport.request({
         method: 'GET',
-        path: `${ML_COMMONS_BASE_API}/memory/trace/${interactionId}/_list`,
+        path: `${ML_COMMONS_BASE_API}/memory/message/${interactionId}/traces`,
       })) as ApiResponse<{
         traces: Array<{
-          conversation_id: string;
-          interaction_id: string;
+          memory_id: string;
+          message_id: string;
           create_time: string;
           input: string;
           response: string;
           origin: string;
-          parent_interaction_id: string;
           trace_number: number;
         }>;
       }>;
 
       return response.body.traces.map((item) => ({
-        interactionId: item.interaction_id,
-        parentInteractionId: item.parent_interaction_id,
```

Comment:
Q: any context why remove `parentInteractionId`?

Reply:
parentInteractionId is not consumed in current Chatbot and remove for reduce confusion.